server/src/resolvers/storeFrontResolver.ts

```diff
@@ -0,0 +1,91 @@
+import { InventoryRepo } from "../repositories/Store/inventoryRepository";
+import { PurchaseOrderRepo } from "../repositories/Store/poRepository";
+import { InventoryService } from "../service/InventoryService";
+import { PurchaseOrderService } from "../service/PurchaseOrderService";
+
+const inventoryService = new InventoryService(new InventoryRepo());
+const purchaseOrderService = new PurchaseOrderService(new PurchaseOrderRepo());
+
+const StorefrontResolvers = {
+    Query: {
+      // InventoryItems: [InventoryItem]
+      InventoryItems: async (_: any, args: any, context: any) => {
+        return inventoryService.getAllInventoryItems();
+      },
+
+      //InventoryItem(Id: ID!): InventoryItem
+      InventoryItem: async(_: any, args: any, context: any) => {
+        return inventoryService.getInventoryItemsById(args.Id);
+      },
+
+      // PurchaseOrders: [PurchaseOrder]
+      PurchaseOrders: async (_: any, args: any, context: any) => {
+        return await purchaseOrderService.getAllPurchaseOrders();
+      },    
+      
+      // PurchaseOrder(Id: ID!): PurchaseOrder
+      PurchaseOrder: async (_: any, args: any, context: any) => {
+        return purchaseOrderService.getPurchaseOrder(args.Id);
+      }, 
+    },
+  
+    Mutation: {
+      // createPurchaseOrder(PO: PurchaseOrderInput): PurchaseOrder  
+      createPurchaseOrder: async (_: any, args: any) => {
```

Comment:
Would this be possible?
``` ts
createPurchaseOrder: async (_: any, args: { PO: PurchaseOrder }): PurchaseOrder => {
```
I like that we have types in the comments above but I think it would be even better if we could write them as actual code.

Reply:
If this works I will 100% do this